nistats/first_level_model.py

```diff
@@ -406,7 +413,7 @@ def fit(self, run_imgs, paradigms=None, confounds=None,
                 start_time = self.slice_time_ref * self.t_r
                 end_time = (n_scans - 1 + self.slice_time_ref) * self.t_r
                 frame_times = np.linspace(start_time, end_time, n_scans)
-                design = make_design_matrix(frame_times, paradigms[run_idx],
+                design = make_design_matrix(frame_times, events[run_idx],
                                             self.hrf_model, self.drift_model,
```

Comment:
I think that I agree with the change from 'paradigms' to 'events', but please try and keep this type of changes for other PRs, since this has exactly no relationship with second-level models.